benchmarks/AuraDiBench.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace PhpBench\Benchmarks\Container;
+
+use Aura\Di\ContainerBuilder;
+
+/**
+ * @Groups({"aura-di"}, extend=true)
+ */
+class AuraDiBench extends ContainerBenchCase
+{
+    private $container;
+
+    public function benchGetUnoptimized()
+    {
+        $this->container->get('bicycle_factory');
+    }
+
+    public function benchGetOptimized()
+    {
+        $this->container->get('bicycle_factory');
+    }
+
+    public function benchGetPrototype()
+    {
+        $this->container->newInstance('PhpBench\Benchmarks\Container\Acme\BicycleFactory');
+    }
+
+    public function benchLifecycle()
+    {
+	$this->init(); 
```

Comment:
Yeah, this tests the cost of instantiating the container and retreving a service.